explorer/package.json

```diff
@@ -7,41 +7,34 @@
   "license": "MIT",
   "keywords": [],
   "scripts": {
-    "predev": "yarn autosetup",
+    "addclnode": "ts-node src/bin/clnodes.ts add",
+    "admin:seed": "ts-node src/bin/admin.ts seed",
+    "depcheck": "echo \"@chainlink/explorer\" && depcheck --ignore-dirs=client || exit 0",
+    "predev": "yarn automigrate",
     "dev": "concurrently \"cd client && yarn start\" \"ts-node-dev --respawn --transpileOnly ./src/index.ts\"",
     "dev:client": "yarn workspace @chainlink/explorer-client run start",
     "dev:server": "ts-node-dev --respawn --transpileOnly ./src/index.ts",
     "prebuild": "yarn install",
     "build": "yarn workspace @chainlink/explorer-client run build",
     "prod": "yarn migration:run && ts-node ./src/index.ts",
-    "pretest": "yarn autosetup",
+    "pretest": "NODE_ENV=test yarn automigrate",
     "test": "yarn jest --runInBand --detectOpenHandles",
-    "pretest-ci": "yarn autosetup",
     "test-ci": "yarn jest src/__tests__ --runInBand --detectOpenHandles",
-    "pretest-ci:e2e": "yarn autosetup",
     "test-ci:e2e": "yarn build && yarn test-ci:e2e:no-build",
-    "pretest-ci:e2e:no-build": "yarn autosetup",
     "test-ci:e2e:no-build": "yarn jest e2e/ --runInBand --detectOpenHandles",
-    "pretest-ci:silent": "yarn autosetup",
     "test-ci:silent": "yarn test-ci --silent",
-    "pretest-ci:e2e:silent": "yarn autosetup",
     "test-ci:e2e:silent": "yarn test-ci:e2e --silent",
     "lint": "eslint --ext .js,.ts,.tsx,.jsx src e2e",
     "lint:fix": "yarn lint --fix",
     "format": "prettier --write \"**/*\"",
     "migration:run": "ts-node ./src/bin/migrator.ts migrate",
     "migration:revert": "ts-node ./src/bin/migrator.ts revert",
-    "test:migration:run": "cross-env TYPEORM_NAME=test ts-node ./src/bin/migrator.ts migrate",
-    "autosetup": "yarn install && yarn automigrate",
-    "automigrate": "md5sum -c .migrations.${TYPEORM_NAME:-dev}.md5 2>/dev/null || yarn migration:run && md5sum src/bin/migrator.ts src/migration/* > .migrations.${TYPEORM_NAME:-dev}.md5",
-    "addclnode": "ts-node src/bin/clnodes.ts add",
-    "admin:seed": "ts-node src/bin/admin.ts seed",
-    "depcheck": "echo \"@chainlink/explorer\" && depcheck --ignore-dirs=client || exit 0"
+    "test:migration:run": "NODE_ENV=test yarn migration:run",
```

Comment:
might need cross-env on here if we plan to support windows developers

Reply:
^